docs/contributor_guide/development_process.rst

```diff
@@ -100,11 +97,11 @@ The Requirements Files
 
 The prerequisites for Fairlearn are split between three separate files:
```

Comment:
Only two files are listed in the bulleted list. Could be confusing to new contributors, is it an error?

Reply:
oh that is interesting. I suppose that there might have been three at some point? It was last edited many years ago :) I have changed it to "two". Thank you for being so thorough!

Reply:
Resolving!